src/python/pants/auth/basic_auth.py

```diff
@@ -10,7 +10,7 @@
 
 from pants.auth.cookies import Cookies
 from pants.subsystem.subsystem import Subsystem
-
+from pants.version import VERSION
 
```

Comment:
The single blank line instead of 2 got you on the lint CI shard. Should go green after.